src/data/inchi-dictionary.ts

```diff
@@ -2,430 +2,825 @@ import { INCHIProps } from '../interfaces/utils';
     molecularWeight: 367.5,
   },
+
   AmmoniumChromate: {
+    CAS: '',
     inchiString: 'InChI=1S/Cr.2H3N.4O/h;2*1H3;;;;/q;;;;;2*-1/p+2',
     inchiKey: 'MFFLHUNPSHBKRG-UHFFFAOYSA-P',
     molecularWeight: 152.07,
   },
+
   AmmoniumCyanide: {
+    CAS: '12211-52-8',
     inchiString: 'InChI=1S/CN.H3N/c1-2;/h;1H3/q-1;/p+1',
     inchiKey: 'ICAIHGOJRDCMHE-UHFFFAOYSA-O',
     molecularWeight: 44.06,
   },
+
   AmmoniumDichromate: {
+    CAS: '7789-09-5',
     inchiString: 'InChI=1S/2Cr.2H3N.7O/h;;2*1H3;;;;;;;/q;;;;;;;;;2*-1/p+2',
     inchiKey: 'JOSWYUNQBRPBDN-UHFFFAOYSA-P',
     molecularWeight: 252.06,
   },
+
   AmmoniumFerrocyanide: {
+    CAS: '',
     inchiString: '',
     inchiKey: '',
     molecularWeight: 392.19,
   },
+
   AmmoniumFluoride: {
+    CAS: '12125-01-8',
     inchiString: 'InChI=1S/FH.H3N/h1H;1H3',
     inchiKey: 'LDDQLRUQCUTJBB-UHFFFAOYSA-N',
     molecularWeight: 37.04,
   },
+
   AmmoniumFluorideAcid: {
+    CAS: '1341-49-7',
     inchiString: 'InChI=1S/2FH.H3N/h2*1H;1H3',
     inchiKey: 'KVBCYCWRDBDGBG-UHFFFAOYSA-N',
     molecularWeight: 57.04,
   },
+
   AmmoniumFormate: {
+    CAS: '540-69-2',
     inchiString: 'InChI=1S/CH2O2.H3N/c2-1-3;/h1H,(H,2,3);1H3',
     inchiKey: 'VZTDIZULWFCMLS-UHFFFAOYSA-N',
     molecularWeight: 63.06,
   },
+
   AmmoniumHydrosulfide: {
+    CAS: '',
     inchiString: 'InChI=1S/H3N.H2S/h1H3;1H2',
     inchiKey: 'HIVLDXAAFGCOFU-UHFFFAOYSA-N',
     molecularWeight: 51.11,
   },
+
   AmmoniumHydroxide: {
+    CAS: '12168-30-8',
     inchiString: 'InChI=1S/H3N.H2O/h1H3;1H2',
     inchiKey: 'VHUUQVKOLVNVRT-UHFFFAOYSA-N',
     molecularWeight: 35.05,
   },
+
   AmmoniumMolybdate: {
+    CAS: '13106-76-8',
     inchiString: 'InChI=1S/Mo.2H3N.4O/h;2*1H3;;;;/q;;;;;2*-1/p+2',
     inchiKey: 'APUPEJJSWDHEBO-UHFFFAOYSA-P',
     molecularWeight: 196.01,
   },
+
   AmmoniumHeptamolybdate: {
+    CAS: '',
     inchiString:
       'InChI=1S/7Mo.6H3N.10H2O.18O/h;;;;;;;6*1H3;10*1H2;;;;;;;;;;;;;;;;;;/q;;;;3*+2;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;/p-6',
     inchiKey: 'FIXLYHHVMHXSCP-UHFFFAOYSA-H',
     molecularWeight: 1235.86,
   },
+
   AmmoniumNitrateAlpha: {
+    CAS: '6484-52-2',
     inchiString: 'InChI=1S/NO3.H3N/c2-1(3)4;/h;1H3/q-1;/p+1',
     inchiKey: 'DVARTQFDIMZBAA-UHFFFAOYSA-O',
     molecularWeight: 80.04,
   },
+
   AmmoniumNitrateBeta: {
+    CAS: '',
     inchiString: '',
     inchiKey: '',
     molecularWeight: 80.04, // Both AmmoniumNitrate alpha and beta have same molecular weight so should we use same inchi string not available as alpha and beta on pubchem
   },
+
   AmmoniumNitrite: {
+    CAS: '13446-48-5',
     inchiString: 'InChI=1S/HNO2.H3N/c2-1-3;/h(H,2,3);1H3',
     inchiKey: 'CAMXVZOXBADHNJ-UHFFFAOYSA-N',
     molecularWeight: 64.04,
   },
+
   AmmoniumOsmochloride: {
+    CAS: '12125-08-5',
     inchiString: 'InChI=1S/6ClH.2H3N.Os/h6*1H;2*1H3;/q;;;;;;;;+4/p-4',
     inchiKey: 'SRBXXQDKBKTWOC-UHFFFAOYSA-J',
     molecularWeight: 439,
   },
+
   AmmoniumOxalate: {
```

Comment:
Ammonium oxalate
CAS : 1113-38-8
Molecular Weight : 124.10
InChI=1S/C2H2O4.2H3N/c3-1(4)2(5)6;;/h(H,3,4)(H,5,6);2*1H3
VBIXEXWLHSRNKB-UHFFFAOYSA-N

Reply:
https://www.sigmaaldrich.com/IN/en/substance/ammoniumoxalatemonohydrate142116009707?context=product

Reply:
Its okay checked molecular formula too in perry its `(NH4)2C2O4⋅H2O`

Reply:
No change needed.